src/patternfly/components/Select/select.scss

```diff
@@ -356,6 +363,20 @@
     pointer-events: none;
     background-color: var(--pf-c-select__menu-item--disabled--BackgroundColor);
   }
+
+  &.pf-m-selected {
```

Comment:
I feel like this is where the focus styles should go? i.e. the background color change